src/screens/Products/ProductDetail.js

```diff
@@ -0,0 +1,266 @@
+import React, { Component } from 'react'
+import {
+  Button,
+  ProfilePicture,
+  Wallpaper,
+  Card,
+  CardSection
+} from '@components'
+import { Text, ScrollView, StyleSheet, LayoutAnimation } from 'react-native'
+import { withMappedNavigationProps } from 'react-navigation-props-mapper'
+import { Icon } from 'react-native-elements'
+
+class ProductDetail extends Component {
+  static navigationOptions = {
+    title: 'Product Detail'
+  };
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      showModel: false,
+      drawerOpen: false,
```

Comment:
I believe rawerOpen in this.state = {} is not being used so possibly remove unless it is planned to be used later on?